### PR TITLE
Feature UI video converter and Automatic download

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/Models/VideoRequestForm.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/Models/VideoRequestForm.java]: You should change the author :)

File: src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java [hunk 2]
```diff
@@ -78,15 +82,15 @@ public String executeRequest(final IRequestForm newRequestForm) throws ClientPro
      */
     public void download(final String filePath) throws IOException {
         String localToken = authGetToken();
-        String sURL = "http://localhost:8080/api/download/img1.png";
+        String sURL = "http://localhost:8080/api/download/";
 
         CloseableHttpClient localHttpClient = HttpClients.createDefault();
         HttpGet request = new HttpGet(sURL);
-        request.setHeader("Authorization", "Bearer " + localToken);
+//        request.setHeader("Authorization", "Bearer " + localToken);
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java]: This comment should be deleted

File: src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java [hunk 3]
```diff
@@ -118,7 +123,8 @@ public String authGetToken() throws IOException {
 
     /**
      * Adds text field to http request.
-     * @param key String with the key
+     *
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java]: This line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java [hunk 4]
```diff
@@ -127,7 +133,8 @@ public void addTextBody(final String key, final String value) {
 
     /**
      * Adds file field to http request.
-     * @param key String with the key
+     *
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java]: This line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java [hunk 5]
```diff
@@ -166,4 +174,50 @@ public void addBodyField(final Parameter parameter) {
             addFileBody(parameter.getKey(), parameter.getValue());
         }
     }
+
+    /**
+     * Download a File by url
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java]: Java docs chould be in third person

File: src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java [hunk 6]
```diff
@@ -166,4 +174,50 @@ public void addBodyField(final Parameter parameter) {
...
+            saveFileFromUrlWithJavaIO(dirName + nameFile, url);
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Download a File by URL
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java]: Java docs chould be in third person

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java [hunk 7]
```diff
@@ -74,6 +74,53 @@ public void setName(final String newName) {
...
+                return spacesForTV;
+            case "Mobile":
+                return spacesForMobile;
+            default:
+                return 0;
+        }
+    }
+
+    /**
+     * Add blank spaces
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java]: Java docs chould be in third person

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 8]
```diff
@@ -19,11 +22,16 @@
 import java.awt.Font;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.io.IOException;
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: This blank line should be deleted

File: logs/propertieslogs.log [hunk 9]
```diff
@@ -954,3 +954,209 @@
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - finish
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - start
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - Execute Try
+[INFO ] 2021-04-18 22:05:23.723 [main] ConverterApplication - start
```
Comment (cristianfchq) [inline logs/propertieslogs.log:957]: I think you shouldn't upload the logs

**Review by danielaSantaCruz — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**